entities/communication/voiceCall.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "Unique identifier for the call"
+    },
+    "duration": {
+      "type": "int",
+      "description": "Duration of the call in seconds"
+    },
+    "created_at": {
+      "type": "timestamp",
+      "description": "Time the call was created"
+    },
+    "updated_at": {
+      "type": "timestamp",
+      "description": "Time the call was last updated"
+    },
+    "status": {
+      "type": "enum",
+      "values": [
+        "incoming call",
+        "answered by staff",
+        "missed",
+        "answered by vonage",
+      ],
+      "description": "Status of the call"
+    },
+    "direction": {
+      "type": "enum",
+      "values": [
+        "inbound/outbound"
+      ],
+      "description": "Whether the call is inbound or outbound"
+    },
+    "staff_uid": {
+      "type": "string",
+      "description": "Unique identifier for the staff member who answered the call"
+    },
+    "client_uid": {
+      "type": "string",
+      "description": "Unique identifier for the client who made the call"
+    },
+    "from_number_id": {
+      "type": "string",
+      "description": "Unique identifier for the number the call was made from"
+    },
+    "to_number_id": {
+      "type": "string",
+      "description": "Unique identifier for the number the call was made to"
+    },
+    "recording": {
```

Comment:
I think you should use recording_url. the name creates an expectation for an object.

Reply:
This is the field name decided by Aviya, who is currently on reserve duty. 
Perhaps you should discuss this with him when he gets back?

Reply:
This is true for all the field name changes requests (except for the numbers)

Reply:
If this is a uid (and by the type I can see it is), its just an oversight of Aviya and its against our standards. Take it with Eliran or Shelly, but I can not approve it this way